### PR TITLE
[KernelGen][Nvidia] Add special_log_ndtr operator with Triton kernel

### PR DESCRIPTION
**Pull request by bwbwzzz**
## Summary
Adds a Triton kernel for `special_log_ndtr`. Computes the log of the standard normal cumulative distribution function log(Phi(x)) for each element of input.

## Testing
- Parametrized tests over pointwise shapes and float32
- Validated against PyTorch reference on device side
- Tested on: Nvidia, Tianshu, Muxi, Ascend, Hygon

## Performance
Test command: `pytest benchmark/test_special_log_ndtr.py --level core` (NVIDIA H20)

| Configuration | Torch Latency (ms) | Gems Latency (ms) | Speedup | TFLOPS |
|---|---|---|---|---|
| pointwise shapes / float32 | — | — | — | — |

## Multi-backend Testing
| Backend | Accuracy Test | Benchmark | Speedup (mean) | Notes |
|---|---|---|---|---|
| Nvidia (H20) | — | — | — | Primary |
| Tianshu | PASS | PASS (14 cases) | 1.764 | — |
| Muxi | PASS | PASS (14 cases) | 1.001 | — |
| Ascend | PASS | N/A | — | — |
| Hygon | FAIL | PASS (14 cases) | 3.916 | RuntimeError on accuracy test |

## Files Changed
- `src/flag_gems/ops/special_log_ndtr.py`: Triton kernel implementation
- `tests/test_special_log_ndtr.py`: Accuracy test
- `benchmark/test_special_log_ndtr.py`: Performance benchmark
- `src/flag_gems/ops/__init__.py`: Register import and `__all__`
- `src/flag_gems/__init__.py`: Register to `_FULL_CONFIG`
- `conf/operators.yaml`: Add operator entry (kind: Math, stage: alpha 5.1)

### CODE REVIEW

**Comment by w1120029931-bit** [comment]
please resolve conflicts

**Review by Dongxu-H — DISMISSED** [review]
## 代码审查：special_log_ndtr

### 🔴 关键问题

**1. 合并冲突需要解决**
此 PR 当前处于 CONFLICTING 状态，需要 rebase 到最新的 master 分支并解决冲突。

**2. 测试容差过大**
```python
utils.gems_assert_close(res_out, ref_out, dtype, atol=0.2)
```
容差 `atol=0.2` 太大了！log_ndtr 函数的输出范围大约在 [-40, 0]，0.2 的绝对误差相当于 20% 以上的误差，这表明实现可能存在数值精度问题。

**建议修复：**
- 对于 x < -10，使用 log_ndtr 的渐近展开：`log_ndtr(x) ≈ -0.5 * x^2 - log(|x|) - 0.5 * log(2π)`
- 对于 x > 6，cdf 接近 1，需要特别处理以避免 log(1-ε) 的精度损失
- 考虑使用 `log1p` 提高精度：`log(0.5 * (1 + erf(x/sqrt(2)))) = -log(2) + log1p(erf(x/sqrt(2)))`

### 🟡 建议改进

**1. 数值稳定性优化**
当前实现：
```python
cdf_value = 0.5 * (1.0 + erf_result)
log_cdf = tl.log(cdf_value)
```

问题：当 x 很大时，erf 接近 1，导致 cdf 接近 1，log(cdf) 精度损失。

**建议改进：**
```python
# 使用 log1p 提高精度
log_cdf = -0.6931471805599453 + tl.math.log1p(erf_result)  # -log(2) + log1p(erf)
```

**2. 负数大值处理**
当 x → -∞，log_ndtr(x) → -∞，但直接计算 erf 会导致精度损失。建议使用互补误差函数 erfc：
```python
# 对于 x < 0 的大负数
log_cdf = tl.math.log(tl.math.erfc(-x * SQRT_HALF) * 0.5)
```

**3. 缺少 in-place 版本的测试**
`special_log_ndtr_` 函数没有对应的测试。

### ✅ 优点

- 实现简洁，使用 Triton 内置 erf 函数
- 代码结构清晰

### 📝 建议的改进实现

```python
@pointwise_dynamic(promotion_methods=[(0, "DEFAULT")])
@triton.jit
def special_log_ndtr_func(x):
    SQRT_HALF = 0.7071067811865476
    LOG_HALF = -0.6931471805599453
    x_fp32 = x.to(tl.float32)
    
    # 使用 log1p 提高精度
    erf_result = tl.math.erf(x_fp32 * SQRT_HALF)
    log_cdf = LOG_HALF + tl.math.log1p(erf_result)
    
    return log_cdf.to(x.dtype)
```

---
请解决合并冲突并改进数值精度后重新提交。

File: src/flag_gems/ops/__init__.py [hunk 1]
```diff
@@ -1107,6 +1108,8 @@
     "special_i0e_out",
     "special_i1",
     "special_i1_out",
+    "special_log_ndtr",
+    "special_log_ndtr_",
```
Comment (tengqm) [inline src/flag_gems/ops/__init__.py:1127]: We do have an inplace version and we don't register it?

File: src/flag_gems/__init__.py [hunk 2]
```diff
@@ -663,6 +663,7 @@ def torch_ge(v):
     ("special_i0e.out", special_i0e_out),
     ("special_i1", special_i1),
     ("special_i1.out", special_i1_out),
+    ("special_log_ndtr", special_log_ndtr),
```
Comment (tengqm) [inline src/flag_gems/__init__.py:680]: here the in-place version is missing?

File: tests/test_special_log_ndtr.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import pytest
+import torch
+
+import flag_gems
+
+from . import accuracy_utils as utils
+
+
+@pytest.mark.special_log_ndtr
```
Comment (tengqm) [inline tests/test_special_log_ndtr.py:9]: We are missing the test for the in-place version?

File: benchmark/test_special_log_ndtr.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import pytest
+import torch
+
+from . import base
+
+
+@pytest.mark.special_log_ndtr
```
Comment (tengqm) [inline benchmark/test_special_log_ndtr.py:7]: We didn't benchmark the in-place version?
